bip-0001.mediawiki

```diff
@@ -67,7 +67,7 @@ Each BIP should have the following parts:
 
 * Abstract -- a short (~200 word) description of the technical issue being addressed.
 
-* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example) or licensed under the Open Publication License.
+* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example).
```

Comment:
Though, I don't see what advantage there is to create another BIP to supersede BIP1, when it is possible to just amend BIP1. This will save a lot of confusion and time.